packages/patternfly-4/react-core/src/components/Button/Button.md

```diff
@@ -59,6 +59,15 @@ LinkButton = () => (
     <Button component="a" isDisabled href="https://pf-next.com/" target="_blank" variant="tertiary">
       Tertiary Link to Core Docs
     </Button>
+    <h2 id="anchor1">Anchor</h2>
```

Comment:
Can you either modify this to have the href be a url or add another example (like the one in the issue).  The issue was opened because a consumer did not understand they needed to have `variant=link` and `component="a"` to get the link to work.